identity/src/main/java/com/stripe/android/identity/viewmodel/IdentityViewModel.kt

```diff
@@ -371,6 +372,16 @@ internal class IdentityViewModel constructor(
      * LiveData for the cause of ErrorScreen.
      */
     val errorCause = MutableLiveData<Throwable>()
+    private val errorCauseObServer = Observer<Throwable> { value -> logError(value) }
```

Comment:
The Payments team has been pretty happy with the switch from `LiveData` to `Flow`. Might be worth considering as well for the Identity SDK.